SwiftTweaks/Tweak.swift

```diff
@@ -76,6 +79,23 @@ extension Tweak where T: SignedNumberType {
 	}
 }
 
+extension Tweak {
+	// This can't be an initializer because generic types apparently can't handle fixed-type initializers
+	public static func stringList(collectionName: String, _ groupName: String, _ tweakName: String, options: [String], defaultValue: String?=nil) -> Tweak<StringOption> {
```

Comment:
This is great! Mind adding a docstring (for example, to explain that the first option in the array will be chosen as default if no `defaultValue` is provided?